identifiers/country-jp/single_member_constituencies.csv

```diff
@@ -0,0 +1,290 @@
+id,name
+ocd-division/country:jp/prefecture:jp-01/ward:1,Constituency 1
+ocd-division/country:jp/prefecture:jp-01/ward:2,Constituency 2
+ocd-division/country:jp/prefecture:jp-01/ward:3,Constituency 3
+ocd-division/country:jp/prefecture:jp-01/ward:4,Constituency 4
+ocd-division/country:jp/prefecture:jp-01/ward:5,Constituency 5
+ocd-division/country:jp/prefecture:jp-01/ward:6,Constituency 6
+ocd-division/country:jp/prefecture:jp-01/district:7,Constituency 7

```

Comment:
Is there significant difference in these consitutency types for Japanese House of Representatives: `ward` `district` `arrondissement`?

These should be updated to all use a consistent type if there's no real difference.

Reply:
Seems like mis-translation of the Wikipedia from Japanese to English. You are right, these are the same.